src/test/java/org/dinosaur/foodbowl/domain/photo/application/PhotoLocalUploaderTest.java

```diff
@@ -0,0 +1,73 @@
+package org.dinosaur.foodbowl.domain.photo.application;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.util.List;
+import org.dinosaur.foodbowl.file.FileTestUtils;
+import org.dinosaur.foodbowl.global.exception.FileException;
+import org.dinosaur.foodbowl.test.IntegrationTest;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.mock.web.MockMultipartFile;
+import org.springframework.web.multipart.MultipartFile;
+
+@SuppressWarnings("NonAsciiCharacters")
+class PhotoLocalUploaderTest extends IntegrationTest {
+
+    @Autowired
+    private PhotoLocalUploader fileSystemPhotoService;
+
+    @Test
+    void 파일을_디렉토리에_저장한다() {
+        MockMultipartFile multipartFile = new MockMultipartFile("bucket", "foodBowl.jpg", MediaType.TEXT_PLAIN_VALUE,
+                "Hello Images".getBytes());
+        List<MultipartFile> multipartFiles = List.of(multipartFile);
+
+        List<String> filePaths = fileSystemPhotoService.upload(multipartFiles, "test");
+
+        assertThat(filePaths).isNotEmpty();
+        FileTestUtils.cleanUp();
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"foodBowl.zip", "foodBowl.pdf", "foodBowl.docx", "foodBowl.pptx"})
```

Comment:
```suggestion
    @ParameterizedTest
    @ValueSource(strings = {"foodBowl.zip", "foodBowl.pdf", "foodBowl.docx", "foodBowl.pptx"})
```
```suggestion
    @ValueSource(strings = {"foodBowl.zip", "foodBowl.pdf", "foodBowl.docx", "foodBowl.pptx"})
    @ParameterizedTest
```
`@Test`가 밑으로 오도록 하는 컨벤션 어떠신가요? ㅎㅎ

Reply:
요것도 정해지면 테스트 전반적으로 확인 부탁드리겠습니다 ✅

Reply:
ValueSource가 아래에 있는게 더 읽기 좋지 않나욥?

어떤 테스트다 -> 어떤 source를 사용할 것이다 처럼 자연스럽게 읽혀서요 !

Reply:
설득당했습니다 :)